### PR TITLE
fix: add secondary wif when using encryption password

### PR DESCRIPTION
**Pull request by alfonsobries**
## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/domains/transaction/hooks/use-wallet-signatory.ts [hunk 1]
```diff
@@ -34,6 +35,15 @@ export const useWalletSignatory = (
 				return wallet.signatory().mnemonic(mnemonic);
 			}
 
+			if (encryptionPassword && secondMnemonic) {
+				await wallet.confirmationWIF().set(secondMnemonic, encryptionPassword);
```
Comment (faustbrian) [inline src/domains/transaction/hooks/use-wallet-signatory.ts:39]: This should be set when a wallet is being imported/created where the user enters their desired password.

Comment (alfonsobries) [reply src/domains/transaction/hooks/use-wallet-signatory.ts:39]: @faustbrian, that means to update the import form for adding the second mnemonic, right? Currently, (unless I missed something) We dont have the second mnemonic until the transfer form

Comment (faustbrian) [reply src/domains/transaction/hooks/use-wallet-signatory.ts:39]: If someone wants to use encryption with a second mnemonic we would need to add an import method for that to store it during import to later on decrypt it during signing.

**Comment by alfonsobries** [comment]
Closing the in favour of a new PR in the Payvo repository

**Pull request closed without merging**